src/components/EnterprisePageComponent.js

```diff
@@ -8,24 +8,45 @@ var slug = require('slug/slug-browser');
 slug.defaults.mode = 'rfc3986';
 
 class EnterprisePageComponent extends React.Component {
+  /**
+   * Set the default state
+   */
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      enterprise: null
+    };
+  }
+
+  componentDidMount() {
+    this.fetchEnterprise();
+  }
+
+  fetchEnterprise() {
+    var component = this,
+      url  = this.context.config.api_root + '/enterprise/' + this.props.params.id;
+
+    fetch(url)
+      .then(function(response) {
+        if (response.ok) {
```

Comment:
Yep, I create issue #60 to handle errors in general. I'll look into that shortly